setup.py

```diff
@@ -1,20 +1,20 @@
-"""Setup script for django-app-namespace-template-loader"""
+"""Setup script for django-apptemplates"""
 import os
 
 from setuptools import setup
 from setuptools import find_packages
 
-__version__ = '0.4'
+__version__ = '2.0'
 __license__ = 'BSD License'
```

Comment:
Former package was MIT licence, but that's supposed to be compatible with BSD so the change can happen silently, and since the whole MIT codebase is wiped, it shouldn't matter anyways.

Source: https://en.wikipedia.org/wiki/License_compatibility#/media/File:Floss-license-slide-image.png